Project-Lily/Project-Lily/NetworkManager.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NetworkManager.swift
+//  Project-Lily
+//
+//  Created by Kendall Poindexter on 11/16/18.
+//  Copyright © 2018 Kendall Poindexter. All rights reserved.
+//
+
+import Foundation
+
+class NetworkManager {
+    
+    private let urlString = "https://private-52aac-breeds1.apiary-mock.com/breeds"
+    
+    func createDogURLSession(completion: @escaping ([Dog]) -> Void) {
```

Comment:
Perhaps this method name could be more descriptive like `fetchDogs` or something? I don't think `createDogURLSession` precisely describes what's going on here.

Reply:
Ah makes since to me. I think fetch dogs is a solid name for this function.

- [ ] Change createDogSession to fetchDogs